doc/esp8266wifi/station-class.rst

```diff
@@ -122,16 +122,16 @@ The following IP configuration may be provided:
    (like e.g. *www.google.co.uk*) and translate them for us to IP
    addresses
 
-For Arduino networking API compatibilty the ESP8266WiFi library supports for IPv4 additional versions of the ``config`` function.
+For Arduino networking API compatibility, the ESP8266WiFi library supports IPv4-only additional versions of the ``config`` function:
 
 .. code:: cpp
 
-    WiFi.config(local_ip)
-    WiFi.config(local_ip, dns)
-    WiFi.config(local_ip, dns, gateway)
+    WiFi.config(local_ip)                        (discouraged, insufficiently accurate, for Arduino API portability)
+    WiFi.config(local_ip, dns)                   (discouraged, insufficiently accurate, for Arduino API portability)
+    WiFi.config(local_ip, dns, gateway)          (discouraged, insufficiently accurate, for Arduino API portability)
     WiFi.config(local_ip, dns, gateway, subnet)
 
-Versions where some of ``dns``, ``gateway`` and ``subnet`` parameters are not specified use a default value. Default ``subnet`` is 255.255.255.0. Default ``gateway`` and ``dns`` are derived from ``local_ip`` by changing the last number to 1. To return to DHCP you can use ``WiFi.config(INADDR_NONE);``.
+Versions where some of ``dns``, ``gateway`` and ``subnet`` parameters are not specified use a default value. Default ``subnet`` is 255.255.255.0. Default ``gateway`` and ``dns`` are derived from ``local_ip`` by changing the last number to 1 (discouraged). To return to DHCP you can use ``WiFi.config(INADDR_NONE);``.
```

Comment:
instead of "(discouraged)" you could write "(discouraged as it may not apply to any local network)"